tensorflow/compiler/mlir/tensorflow/BUILD

```diff
@@ -264,6 +264,21 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "unroll_batch_matmul_pass",
+    srcs = [
+        "transforms/unroll_batch_matmul.cc",
+    ],
+    hdrs = [
+        "transforms/unroll_batch_matmul.h",
+    ],
+    deps = [
+        ":tensorflow",
+        "@com_google_absl//absl/memory",
+        "@llvm-project//mlir:QuantOps",
```

Comment:
:+1: Thanks for the fast review.

I removed it in c27b0d18ea685a8834a9594cf6e29c2daa6f042b, I only added it because it was copied over in 79aab7dd7b5864e223183d040a0d89ddf7865bb3.

Reply:
Looks like this was still used somewhere. I'll check if I can reproduce this locally.